### PR TITLE
Forward purchaseUnavailableRegion flag to subscriptions iframe

### PR DESCRIPTION
**Pull request by fikus**
Internal ref: b/228193791

### CODE REVIEW

**Comment by jpettitt** [comment]
Do we need to add this to the button rendering case where it's button->purchase flow with no offers frame?

File: src/model/auto-prompt-config.js [hunk 1]
```diff
@@ -127,18 +127,27 @@ export class ImpressionConfig {
...
 /**
- * Predicates of whether or not to show button and prompt.
+ * Predicates to control UI elements.
  */
+// TODO(b/228856600): Move to a more general location.
 export class UiPredicates {
   /**
    * @param {boolean|undefined} canDisplayAutoPrompt
    * @param {boolean|undefined} canDisplayButton
+   * @param {boolean|undefined} purchaseUnavailableRegion
```
Comment (adolgachev) [inline src/model/auto-prompt-config.js:137]: I think 3 is probably the limit on this but should be okay and not worth changing to object right now. Can you please file another bug (or merge into above) to change to object args though for later?

**Review by adolgachev — APPROVED** [review]

File: src/model/auto-prompt-config.js (hunk 1 above)
Comment (fikus) [reply src/model/auto-prompt-config.js:137]: Filed b/229131111.

**Comment by fikus** [comment]
@jpettitt I added you to a thread on the PRD.

**Pull request merged**